torch/onnx/symbolic_opset9.py

```diff
@@ -931,6 +931,12 @@ def selu(g, input):
 
 @parse_args('v', 'i', 'v')
 def index_select(g, self, dim, index):
+    if index.type().kind() == "CompleteTensorType" or index.type().kind() == "DimensionedTensorType":
```

Comment:
We could avoid relying on the tensor type here and use _maybe_get_const() instead.

index_const = sym_help._maybe_get_const(index, 'i')
    if not sym_help._is_value(index_const):
        index = g.op("Reshape", index, g.op("Constant", value_t=torch.LongTensor([-1])))